gunicorn/arbiter.py

```diff
@@ -509,6 +516,7 @@ def spawn_worker(self):
             util._setproctitle("worker [%s]" % self.proc_name)
             self.log.info("Booting worker with pid: %s", worker_pid)
             self.cfg.post_fork(self, worker)
+            os.environ['INSTANCE'] = str(worker_id)
```

Comment:
If INSTANCE is not part of a standard or a widely used best practice, I'd rename it to GUNICORN_WORKER_INSTANCE or GUNICORN_INSTANCE.